src/main/scala/io/scala/data/SpeakersInfo.scala

```diff
@@ -9,8 +9,11 @@ object SpeakersInfo:
   val jimNewton = Speaker(
     name = "Jim Newton",
     photo = Some("jim.webp"),
-    job = "",
-    company = "EPITA Rennes",
+    description = """|Jim Newton is a junior researcher professor at EPITA (Paris), working in the automata research group at EPITA Research Lab.  He holds a PhD in Computer Science from Sorbonne Université.
+                     |Jim has 35+ years of experience in professional Lisp programming in the IC/VLSI design sector, at Cypress Semiconductor, Intel, and Cadence Design Systems.
+                     |Jim has been programming off and on in Scala for about 5 years.""",
```

Comment:
List ? btw you might be missing a stripmargin, not sure the '|' will be removed.